test-suite/run_djinni.sh

```diff
@@ -90,6 +90,7 @@ fi
     --yaml-prefix "test_" \
     \
     --idl "$in_relative" \
+    --idl-include-path "./djinni/vendor" \
```

Comment:
This is a nit, but I'd leave off the "./" here to be consistent with the way that other relative paths are expressed in this command-line.  You can see the impact that in the inconsistent look of the paths of inFileList.txt